File: client/src/main/java/com/agt/bsuirgek/client/Object/Teacher.java
package com.agt.bsuirgek.client.Object;

import java.util.Map;

public class Teacher {

    public String surname;
    public String name;
    public String patronymic;
    public String data;
    public String phone;
    private Map<String, String> map;

    public Teacher(Map<String, String> map){
        this.map = map;
        surname = map.get("surname");
        name = map.get("name");
        patronymic = map.get("patronymic");
        data = map.get("data");
        phone = map.get("phone");
    }
}

File: server/src/main/kotlin/com/agt/bsuirgek/server/model/Teacher.kt
package com.agt.bsuirgek.server.model

import org.jetbrains.exposed.dao.IntIdTable

class Teacher(map: Map<String, String>) {
    val surname = map["surname"]
    val name = map["name"]
    val patronymic = map["patronymic"]
    val data = map["data"]
    val phone = map["phone"]
    override fun toString() = "Teacher($surname;$name;$patronymic;$data;$phone)"
}

object Teachers : IntIdTable() {
    val surname = varchar("surname", Int.MAX_VALUE)
    val name = varchar("name", Int.MAX_VALUE).nullable()
    val patronymic = varchar("patronymic", Int.MAX_VALUE).nullable()
    val data = varchar("data", Int.MAX_VALUE).nullable()
    val phone = varchar("phone", Int.MAX_VALUE).nullable()
}